cmp/ag-grid/AgGrid.ts

```diff
@@ -141,15 +148,20 @@ class AgGridLocalModel extends HoistModel {
         }
     }
 
+    getRowHeight = ({node}) => {
+        const {sizingMode} = this.model,
+            {groupDisplayType} = this.componentProps,
+            AgGridCmp = AgGrid as any;
+        return node.group && groupDisplayType === 'groupRows'
```

Comment:
This `groupDisplayType` is not familiar to me - could we can add a small comment to explain this additional check?

Reply:
also not familiar -- just copied from the Grid getRowHeight function. Will look into it post merge